static/js/sefaria/sefaria.js

```diff
@@ -2779,6 +2781,56 @@ _media: {},
     return Sefaria.topic_toc.filter(x => x.slug == slug).length > 0;
   },
   sheets: {
+    getSheetsByRef: function(srefs, callback) {
+        return Sefaria._cachedApiPromise({
+          url: `${Sefaria.apiHost}/api/sheets/ref/${srefs}?include_collections=1&include_first_comment=1`,
+          key: srefs,
```

Comment:
Include `include_collections` on the key so that these results remain separate from other results that don't have collections.